build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -538,8 +538,23 @@ message Command {
   }
 
   // The arguments to the command. The first argument must be the path to the
-  // executable, which must be either a relative path, in which case it is
-  // evaluated with respect to the input root, or an absolute path.
+  // executable, which is interpreted as follows:
+  //
+  // - If the path contains no path separators, the path of the executable is
+  //   resolved using the operating system's equivalent of the PATH environment
+  //   variable. Workers are not required to accept shell builtins or functions.
+  // - If the path is relative, it is resolved relative to the working directory
+  //   of the action.
+  // - If the path is absolute, it is resolved relative to the operating
+  //   system's equivalent of the root directory.
```

Comment:
I would drop this sentence- absolute paths are already resolved, and windows doesn't have a root directory.

Reply:
I added this sentence, because I wanted to make it explicit that paths are allowed to be absolute. Maybe the second part of the sentence can be improved? What are your thoughts?

Reply:
How about something like this:

"The first argument specifies the command to run, which may be either an absolute path, a path relative to the working directory, or an unqualified path (without path separators) which will be resolved using the operating system's equivalent of the PATH environment variable."

Reply:
I processed this change two weeks ago, but forgot to mark it as resolved. Doing this retroactively. :-)